pkg/util/util.go

```diff
@@ -144,3 +145,16 @@ func IsReadOnlyDisk(disk *v1.Disk) bool {
 
 	return isReadOnlyCDRom
 }
+
+func AlignImageSize(size int64, logger *log.FilteredLogger) int64 {
```

Comment:
Also - is it possible that in the future we would need to change the alignment size (4k to something else)?
Maybe we can call this function `AlignImageSizeTo4k()` or get the alignment size as a parameter?

Not sure about it, just a thought.

Reply:
Good point, I'll just change the function name for now, and if we ever need to align to any other value we can generalize then.
Thank you for the review!